csaf_2.1/prose/edit/src/conformance.md

```diff
@@ -60,6 +60,7 @@ The entities ("conformance targets") for which this document defines requirement
 
 A text file or data stream satisfies the "CSAF document" conformance profile if it:
 
+* satisfies to the syntax and semantics defined in section [sec](#date-time)
```

Comment:
Nit: either `satisfies the` or `conforms to the` - and maybe not vary. Why not conforms to all over the place (instead of adding satisfaction)?

Reply:
I just copied from ~above~ below...

Reply:
I guess we used the `satisfies` as we were talking about requirements (i.e. for the "tool" conformance targets) so that when we added the CSAF document target it felt still okay-ish

Reply:
My bad - you are right.